btree/btree.c

```diff
@@ -5549,6 +5558,7 @@ int64_t btree_create_tree_tick(struct m0_sm_op *smop)
 		 */
 		if (!addr_is_aligned(data->addr))
 			return M0_ERR(-EFAULT);
+#endif
```

Comment:
Since this check has been disabled, the condition in UT basic_tree_op_icp also needs to be commented out. This condition is from lines 7553 to 7556.